bodhi/tests/server/services/test_graphql.py

```diff
@@ -92,3 +92,64 @@ def test_enumfields(self):
                 }]
             }
         }
+
+    def test_getRelease(self):
+        """Testing getRelease."""
+        client = Client(schema)
```

Comment:
We could use a setup_method in the TestGraphQLService class to declare the clien and release variable (see https://github.com/fedora-infra/bodhi/blob/develop/bodhi/tests/server/test_config.py#L55). That way you don't have to copy/paste that to every tests and you can use self.client and self.release.